src/org/zaproxy/zap/extension/ascanrulesAlpha/resources/help/contents/ascanalpha.html

```diff
@@ -52,22 +52,36 @@ <H2>Httpoxy</H2>
 <b>IMPORTANT</b> - the computer that ZAP is running on must accept incoming requests on arbitrary ports -
 if a firewall prevents incoming connections then this rule will not work. 
 
-<H2>HTTPS As HTTP Scanner</H2>
+<H2>HTTPS Content Available via HTTP</H2>
 This active scanner attempts to access content that was originally accessed via HTTPS (SSL/TLS) via HTTP.
 
 <H2>LDAP Injection</H2>
 LDAP Injection may be possible. It may be possible for an attacker to bypass authentication controls, and to view and modify arbitrary data in the LDAP directory.
 
-<H2>SQL Injection - MsSQL</H2>
-This active scanner attempts to inject MsSQL specific sleep commands into parameter values and analyzes the server's response time to see if the sleep is effectively executed on the server (indicating a successful SQL injection attack).
+<H2>Proxy Disclosure</H2>
+Attempts to detect and fingerprint proxy server(s). This information helps a potential attacker to determine.<br>
+- A list of targets for an attack against the application.<br>
```

Comment:
Any reason not to use <ul><li>... for these?

Reply:
Nope. Will change it.